content/projects/understanding-loops/_index.md

```diff
@@ -119,9 +124,12 @@ Hint: You have already defined some functions that would be useful in finding th
 
 ### customers who have OPEN baskets
 
-Write a function called `get customers with open baskets` that returns a list/array of email addresses for customers who have baskets that are open.
+Write a function called `get customers with open baskets` that takes in the data array as an argument and returns a list/array of email addresses for customers who have baskets that are open.
+e.g. `getCustomersWithOpenBaskets(dataArray);`
+
 ## Notes to reviewers
 
 - the code must be DRY
 - there can be some global constants to help prevent typos. eg `const DELIVERED="DELIVERED"`
-- functions are supposed to return very specific things
\ No newline at end of file
+- functions are supposed to return very specific things
+- All the functions stated above should take in at least one argument, the data array, and should work with similar data.
```

Comment:
This point starts with a capital letter. All the others are small letters. 
Should they all start with capital letters or should they all be small letters?